sklearn/sklearn-template/template/scripts/train.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Convenience script for running ML training jobs.
+#
+# Prerequisites:
+#   - Google Cloud SDK
+#
+# Globals:
+#   PROJECT_ID: Google Cloud project to use.
+#   BUCKET_ID: Google Cloud Storage bucket to store output.
+#
+# Arguments:
+#  $1: Path or BigQuery table to dataset for ML training and eval
+#  $2: Whether to run `local` (on-prem) or `remote` (GCP)
+
+
+INPUT=$1
+RUN_ENV=$2
+
+if [[ ! "$RUN_ENV" =~ ^(local|remote)$ ]]; then
+  RUN_ENV=local;
+fi
+
+NOW="$(date +"%Y%m%d_%H%M%S")"
+PROJECT_NAME="sklearn_template"
+
+JOB_NAME="${PROJECT_NAME}_training_${NOW}"
+JOB_DIR="gs://$BUCKET_ID/$JOB_NAME"
```

Comment:
As mentioned above, if $BUCKET_ID is not configured, the JOB_DIR would "gs:///$JOB_NAME", which will cause error. I think it's better to make it as a parameter.

Reply:
Ah, I think I understand what the issue was. I'll add some conditionals to guard against undefined `BUCKET_ID`.